app/models/user.py

```diff
@@ -57,10 +57,11 @@ class User(UserMixin, db.Model):
     def __init__(self, **kwargs):
         super(User, self).__init__(**kwargs)
         if self.role is None:
-            if self.email == current_app.config['ADMIN_EMAIL']:
+            if not User.query.first():
+                Role.insert_roles()
                 self.role = Role.query.filter_by(
                     permissions=Permission.ADMINISTER).first()
-            if self.role is None:
+            else:
```

Comment:
@Huntrr Sweet, good work. Last question: do you see why this wasn't an `if-else` before?